app/components/appeals-app-services/locals.tf

```diff
@@ -40,16 +40,17 @@ locals {
         USE_SECURE_SESSION_COOKIES                 = false
       }
     }
+
     #====================================
     # Backends
     #====================================
 
     appeals_service_api = {
       app_name                        = "appeals-api"
       app_service_private_dns_zone_id = var.app_service_private_dns_zone_id
-      endpoint_subnet_id              = var.endpoint_subnet_id
+      endpoint_subnet_id              = var.environment != "dev" ? var.endpoint_subnet_id : null
```

Comment:
We should have a new variable for this! Never good to rely on checking for the environment in multiple places.

Reply:
Variable should be a boolean and be listed in the dev.hcl file. Something like `private_endpoints_enabled` maybe. You can set a default value of true wherever we use it.

Reply:
Addressed in [ecb5fec](https://github.com/Planning-Inspectorate/infrastructure-environments/pull/82/commits/ecb5fecc6ae621ae23a3e954741666aa9137ddc9), pending review.